### PR TITLE
Use camelCase for configuration instead of snake_case, add basic configuration validation

### PR DESCRIPTION
**Pull request by lencioni**

### CODE REVIEW

File: lib/Configuration.js [hunk 1]
```diff
@@ -102,8 +188,8 @@ export default class Configuration {
    */
   get(key, { fromFile } = {}) {
     return this.configs.find((config) => {
-      const appliesTo = config.applies_to || '**';
-      const appliesFrom = config.applies_from || '**';
+      const appliesTo = config.appliesTo || '**';
+      const appliesFrom = config.appliesFrom || '**';
```
Comment (trotzig) [inline lib/Configuration.js]: we could deconstruct these in the method declaration, right?

`({ appliesTo, appliesFrom }) => {...}`

File: lib/Importer.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default class Importer {
     this.editor = new CommandLineEditor(lines);
     this.config = new Configuration(pathToCurrentFile);
     this.pathToCurrentFile = pathToCurrentFile;
-    this.messages = [];
+    this.messages = Array.from(this.config.messages);
```
Comment (trotzig) [inline lib/Importer.js]: Nice way to solve the issue of getting warnings to clients.

**Comment by trotzig** [comment]
Thanks!

**Comment by trotzig** [comment]
I'm going to check the travis output to make sure that tests were actually run. See https://github.com/Galooshi/import-js/pull/257 for context.

**Comment by trotzig** [comment]
Yeah, it looks like tests haven't been run. :(

File: lib/Configuration.js (hunk 1 above)
Comment (lencioni) [inline lib/Configuration.js]: Yeah, and we could probably also provide defaults

```js
({ appliesTo = '**', appliesFrom = '**' }) => {...}
```

Comment (lencioni) [inline lib/Configuration.js]: Oh, but we actually use the `config` object later. We can still simplify this a bit with some destructuring in the function body. I'll add a commit.

**Comment by lencioni** [comment]
I've rebased and plan to merge this once Travis is green.

**Pull request merged**